docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Install the microservice Helm chart 
+title: Loki Microservice Helm chart 
 menuTitle: Install microservice Loki
-description: Installing Loki in microservice (distributed) mode using the Helm chart.
+description: Installing Loki in microservice mode using the Helm chart.
 weight: 300
 keywords: 
 ---
 
-# Install the microservice Helm chart
+# Loki Microservice Helm chart
```

Comment:
This heading needs to revert back to the original to 1) match our Style Guide and 2) match the other installation headings (see the generated TOC to see what I mean).